cfme/tests/ssui/test_ssui_myservice.py

```diff
@@ -27,6 +27,7 @@
 ]
 
 
+@pytest.mark.rhv1
```

Comment:
Here we also need to discuss if SSUI is part of sanity. I would consider it T2, but Id' like to hear arguments for T1

Reply:
I marked it as T1 since Moran (the PM) mentioned it was considered to be an option that is considered to be a replacement to the RHV user portal, but we can discuss this, sure.